### PR TITLE
bpo-23952: Document cgi module's maxlen variable

### PR DESCRIPTION
**Pull request by hugovk**
Using the text suggested in https://bugs.python.org/issue23952.

I think this documentation-only change can skip a NEWS file.

https://bugs.python.org/issue23952

### CODE REVIEW

**Review by rhettinger — CHANGES_REQUESTED** [review]
This is a good start.  Please also update the main docs at `Docs/library/cgi.rst`.

**Comment by hugovk** [comment]
> This is a good start. Please also update the main docs at `Docs/library/cgi.rst`.

Updated!

I have made the requested changes; please review again.

File: Lib/cgi.py [hunk 1]
```diff
@@ -13,6 +13,11 @@
 
 This module defines a number of utilities for use by CGI scripts
 written in Python.
+
+The maxlen variable can be set to an integer indicating the maximum size of a
```
Comment (ethanfurman) [inline Lib/cgi.py]: Change "The maxlen variable" to "The global variable maxlen".

**Review by ethanfurman — CHANGES_REQUESTED** [review]

File: Lib/cgi.py (hunk 1 above)
Comment (hugovk) [reply Lib/cgi.py]: Done!

**Comment by hugovk** [comment]
I have made the requested changes; please review again

**Review by ethanfurman — APPROVED** [review]

**Review by ethanfurman — APPROVED** [review]

**Comment by hugovk** [comment]
`test_issue16464 (test.test_urllib2.MiscTests) ... ERROR` failed with `urllib.error.HTTPError: HTTP Error 404: Not Found`.

<details>
<summary>Details</summary>

```
test_issue16464 (test.test_urllib2.MiscTests) ... ERROR
======================================================================
ERROR: test_issue16464 (test.test_urllib2.MiscTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/contextlib.py", line 155, in __exit__
    self.gen.throw(typ, value, traceback)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/test/support/socket_helper.py", line 245, in transient_internet
    yield
    ^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/test/test_urllib2.py", line 1803, in test_issue16464
    opener.open(request, "1234567890".encode("us-ascii"))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/urllib/request.py", line 525, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/urllib/request.py", line 634, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/urllib/request.py", line 563, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/home/dje/cpython-buildarea/pull_request.edelsohn-rhel-z.refleak/build/Lib/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
urllib.error.HTTPError: HTTP Error 404: Not Found
```

</details>

https://buildbot.python.org/all/#/builders/217/builds/431/steps/5/logs/stdio

Likely unrelated to this docs change, so rescheduling.

**Comment by hugovk** [comment]
Same thing. The `test_issue16464` failure is unrelated; https://bugs.python.org/issue46648 / https://github.com/python/cpython/pull/31146 / https://github.com/python/cpython/pull/31148.

**Pull request merged**